### PR TITLE
paypal recurring tweaking

### PR DESCRIPTION
**Pull request by zamandevs**

### CODE REVIEW

File: lib/gateway/paypal.php [hunk 1]
```diff
@@ -310,6 +317,10 @@ public function paypal_success() {
             $custom    = json_decode( stripcslashes( $postdata['custom'] ) );
             $coupon_id = isset( $custom->coupon_id ) ? $custom->coupon_id : false;
 
+            if ($postdata['period3']) {
```
Comment (hasinur1997) [inline lib/gateway/paypal.php:320]: @zamandevs Use space like `if ( $postdata['period3'] )`

File: lib/gateway/paypal.php [hunk 2]
```diff
@@ -310,6 +317,10 @@ public function paypal_success() {
...
+                update_user_meta($custom->user_id, '_wpuf_used_trial', 'yes');
```
Comment (hasinur1997) [inline lib/gateway/paypal.php:321]: Use spae `update_user_meta( $custom->user_id, '_wpuf_used_trial', 'yes' )`

File: class/transactions-list-table.php [hunk 3]
```diff
@@ -208,16 +214,18 @@ public function prepare_items() {
...
 
-        $this->set_pagination_args( [
-            'total_items' => $total_items,
-            'per_page'    => $per_page,
-        ] );
+        $this->set_pagination_args(
+            [
+				'total_items' => $total_items,
+				'per_page'    => $per_page,
+			]
```
Comment (tareq1988) [inline class/transactions-list-table.php:227]: Indentation problem.

**Comment by tareq1988** [comment]
Please use editorconfig to fix the spacing problem on both the files.

**Pull request merged**